### PR TITLE
Added complete dashboard localization

### PR DESCRIPTION
**Pull request by adagioajanes**
# Description

**Don't worry! This PR looks alot larger than it really is. Most of the changed lines are just formatting cleanup to fit the standard that appears to be in use.**

I was getting ready to try and do some things to the dashboard, but I noticed that it is missing localization in several places. I have added that localization and tried to reuse existing phrases as much as possible. But there are still a few phrases that I had to add.

Because there is so much noise generated by the white space cleanup in the `resources\lang\en\general.php` file, I will open code reviews on the lines that were actually added.

Also, because this adds new phrases, I am marking it as a breaking change. But please feel free to correct me. ;)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [X] Opened the dashboard in a new installation to ensure the welcome message is properly displayed
- [X] Checked the additionally added phrases to make sure they are rendering correctly.

**Test Configuration**:
* PHP version: 7.4.27
* MySQL version: 5.7.36
* Webserver version: PHPStorm Internal Web Server
* OS version: Windows 10 WSL

# Checklist:

- [X] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [X] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [X] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: resources/lang/en/general.php [hunk 1]
```diff
@@ -1,244 +1,254 @@
...
+    'asset'                 => 'Asset',
     'asset_report'          => 'Asset Report',
-    'asset_tag'				=> 'Asset Tag',
-    'assets_available'		=> 'assets available',
-    'audit'				    => 'Audit',
-    'audit_report'			=> 'Audit Log',
-    'assets'				=> 'Assets',
+    'asset_tag'             => 'Asset Tag',
+    'assets_available'      => 'assets available',
+    'assigned'              => 'Assigned',
```
Comment (adagioajanes) [inline resources/lang/en/general.php:23]: `assigned` is a new phrase

File: resources/lang/en/general.php [hunk 2]
```diff
@@ -1,244 +1,254 @@
...
+    'clear_selection'       => 'Clear Selection',
+    'companies'             => 'Companies',
+    'company'               => 'Company',
+    'component'             => 'Component',
+    'components'            => 'Components',
+    'complete'              => 'Complete',
+    'consumable'            => 'Consumable',
+    'consumables'           => 'Consumables',
+    'collapse'              => 'Collapse',
+    'count'                 => 'Count',
```
Comment (adagioajanes) [inline resources/lang/en/general.php:58]: `collapse` and `count` are new phrases

File: resources/lang/en/general.php [hunk 3]
```diff
@@ -1,244 +1,254 @@
...
+    'updated_at'            => 'Updated at',
+    'currency'              => '$', // this is deprecated
+    'current'               => 'Current',
     'custom_report'         => 'Custom Asset Report',
-    'dashboard'				=> 'Dashboard',
-    'days'      			=> 'days',
+    'dashboard'             => 'Dashboard',
+    'dashboard_60_percent'  => '60 % Complete (warning)',
+    'dashboard_nothing_added_message' => 'It looks like you haven&apos;t added anything yet, so we don&apos;t have anything awesome to display. Get started by adding some assets, accessories, consumables, or licenses now!',
+    'dashboard_welcome_message' => 'This is your dashboard. There are many like it, but this one is yours.',
```
Comment (adagioajanes) [inline resources/lang/en/general.php:72]: `dashboard_60_percent`, `dashboard_nothing_added_message`, and `dashboard_welcome_message` are new phrases.

File: resources/lang/en/general.php [hunk 4]
```diff
@@ -1,244 +1,254 @@
...
+    'moreinfo'              => 'More Info',
+    'name'                  => 'Name',
+    'next'                  => 'Next',
+    'next_audit_date'       => 'Next Audit Date',
+    'last_audit'            => 'Last Audit',
+    'new'                   => 'new!',
+    'new_asset'             => 'New Asset',
+    'new_accessory'         => 'New Accessory',
+    'new_consumable'        => 'New Consumable',
+    'new_license'           => 'New License',
```
Comment (adagioajanes) [inline resources/lang/en/general.php:169]: `new_asset`, `new_accessory`, `new_consumable`, and `new_license` are new phrases

File: resources/views/dashboard.blade.php [hunk 5]
```diff
@@ -156,34 +156,34 @@
 
                             <div class="progress">
                                 <div class="progress-bar progress-bar-yellow" role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100" style="width: 60%">
-                                    <span class="sr-only">60% Complete (warning)</span>
```
Comment (adagioajanes) [inline resources/views/dashboard.blade.php:159]: I can't actually get this particular line to display. I am guessing it only shows in some sort of compatibility mode when the html doesn't render correctly. But, I turned it into a phrase anyway.

**Review by adagioajanes — COMMENTED** [review]
Added comments to the phrases that were added. Additionally, added a comment about the dashboard 60 percent message on the blade template.

File: resources/views/dashboard.blade.php (hunk 5 above)
Comment (snipe) [reply resources/views/dashboard.blade.php:159]: `sr-only` means `screen reader only` - it's not meant to show up on the screen.

**Comment by snipe** [comment]
I'd probably check that this PR doesn't duplicate the work that @nuraeil has been working on for the past month or two in #10264

**Comment by adagioajanes** [comment]
Hmmm, it looks like that PR is much further along in the effort. I will link this PR there and ask if any of my changes should be incorporated.

**Pull request closed without merging**